src/data/nat/prime/min_fac.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2019 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import data.nat.prime
+import tactic.fin_cases
+
+/-!
+# Lemmas about min_fac
+
+These lemmas don't work in `data/nat/prime.lean`, because they use `fin_cases` in the proofs,
+which would introduce a cyclic import.
+-/
```

Comment:
Maybe it would be generally useful to add
```lean
lemma two_le_iff : ∀ {n : ℕ}, 2 ≤ n ↔ (n ≠ 0 ∧ n ≠ 1)
| 0 := by norm_num
| 1 := by norm_num
| (n+2) := ⟨λ _, by split; intro h; cases h, λ _, le_add_left _ _⟩
```

Reply:
Maybe this should go in a separate PR now?